src/pages/scientificServices/pipelines/widgets/QuotaRemainingWidget.tsx

```diff
@@ -93,16 +93,16 @@ export const QuotaRemainingWidget = ({ selectedPipeline }: { selectedPipeline?:
       )}
       <div style={{ marginTop: '1rem' }}>
         <a
-          href={`mailto:${SCIENTIFIC_SERVICES_SUPPORT_EMAIL}?subject=Imputation%20Quota%20Increase%20Request`}
+          href={`mailto:${SCIENTIFIC_SERVICES_SUPPORT_EMAIL}?subject=Request%20a%20quote%20for%20quota`}
           style={{ color: '#46A3E9', textDecoration: 'underline', fontWeight: 'bold' }}
         >
-          Apply
+          Request
```

Comment:
super minor, and you're welcome to disregard if you disagree, but to me it reads better if the hyperlink text is "Request a quote" rather than just "Request"

Reply:
updated in https://github.com/DataBiosphere/terra-ui/pull/5419/commits/b61faa313e3b63da321079366379003722293aef !